src/Cache/Http/InvalidateCacheTags.php

```diff
@@ -90,7 +90,7 @@ public function __construct(
     /**
      * {@inheritDoc}
      */
-    public function setEnvironment(EnvironmentInterface $environment): InvalidCacheTags
```

Comment:
It has to return an instance of `InvalidCacheTagsInterface`

Reply:
See above, should get dropped - the service should be stateless.

Reply:
fixed

Reply:
fix naming